### DISCUSSION TITLE
Execute npm init @eslint/config and report an error

### DISCUSSION THREAD

**Discussion by blackawn** [eslint/eslint#17674] [Q&A]
The error is reported as follows: 😆

```sh
'create-config' is not recognized as an internal or external command or operable program
or batch file.
```

node: v18.18.2
npm: 10.2.1

**Comment by Rec0iL99** [comment]
Hi @blackawn, we would need more info to debug this. Did you get the above error when you ran `npm init @eslint/config` for the first time?

**Reply by blackawn** [reply]
Thank you for your reply
I have solved it, it should be my network problem, downloading incomplete data.
I **deleted the** relevant files in the```{user_path}\AppData\Local\npm-cache\_npx``` path and then re-executed it.
😂
